pkg/apis/clouddns/v1/zone_types.go

```diff
@@ -0,0 +1,74 @@
+package v1
+
+import "time"
+
+type Revision struct {
+	CreatedAt  time.Time `json:"created_at"`
+	Identifier string    `json:"identifier"`
+	ModifiedAt time.Time `json:"modified_at"`
+	Records    []Record  `json:"records"`
+	Serial     int       `json:"serial"`
+	State      string    `json:"state"`
+}
+
+type DNSServer struct {
+	// Required - DNS Server name (FQDN).
+	Server string `json:"server"`
+
+	// DNS Server alias
+	Alias string
```

Comment:
is this supposed to be json-encoded as `Alias`?

Reply:
Yes. Was actually already missed in the legacy client code...